helpers/setup_next_run.py

```diff
@@ -37,7 +38,7 @@
 import argparse
 
 global verbose
-verbose=False
+verbose=True
```

Comment:
hmmm, the default should really be set by argparse at the bottom... does this work?

Reply:
I can't get it to work, but that is probably just my poor bash skills. I've removed the hardcoded lines for now.